### PR TITLE
[network] easy rename

### PR DESCRIPTION
**Pull request by zekun000**
## Motivation

ConnectionStatusNotification -> ConnectionNotification
ConnectionHandlerRequest -> TransportRequest
ConnectionNotification -> TransportNotification

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: network/src/peer_manager/mod.rs [hunk 1]
```diff
@@ -221,7 +221,7 @@ where
     /// Address to listen on for incoming connections.
     listen_addr: NetworkAddress,
     /// Connection Listener, listening on `listen_addr`
-    connection_handler: Option<ConnectionHandler<TTransport, TSocket>>,
+    connection_handler: Option<TransportHandler<TTransport, TSocket>>,
```
Comment (davidiw) [inline network/src/peer_manager/mod.rs:224]: but look your struct property's name sucks

File: network/src/peer_manager/conn_status_channel.rs [hunk 2]
```diff
@@ -8,13 +8,13 @@
 //! It provides an mpsc channel which has two ends `conn_status_channel::Receiver`
```
Comment (phlip9) [inline network/src/peer_manager/conn_status_channel.rs:8]: let's rename `conn_status_channel` to `conn_notif_channel` or something

File: network/src/peer_manager/mod.rs [hunk 3]
```diff
@@ -293,7 +293,7 @@ where
         // rethink how we init the PeerManager so we don't have to do this funny thing.
         let connection_handler_notifs_tx = connection_notifs_tx.clone();
         let (connection_handler, listen_addr) = executor.enter(|| {
```
Comment (phlip9) [inline network/src/peer_manager/mod.rs:295]: rename these vars

File: network/src/peer_manager/mod.rs [hunk 4]
```diff
@@ -490,11 +490,11 @@ where
     }
 
     fn start_connection_listener(&mut self) {
```
Comment (phlip9) [inline network/src/peer_manager/mod.rs:492]: start_transport_listener?

File: network/src/peer_manager/mod.rs [hunk 5]
```diff
@@ -713,19 +713,19 @@ where
...
     TTransport: Transport,
     TSocket: AsyncRead + AsyncWrite,
 {
     /// [`Transport`] that is used to establish connections
     transport: TTransport,
     listener: Fuse<TTransport::Listener>,
-    dial_request_rx: channel::Receiver<ConnectionHandlerRequest>,
-    connection_notifs_tx: channel::Sender<ConnectionNotification<TSocket>>,
+    dial_request_rx: channel::Receiver<TransportRequest>,
+    connection_notifs_tx: channel::Sender<TransportNotification<TSocket>>,
```
Comment (phlip9) [inline network/src/peer_manager/mod.rs:725]: transport_notifs_tx

File: network/src/peer_manager/mod.rs [hunk 6]
```diff
@@ -239,13 +239,13 @@ where
...
-    dial_request_tx: channel::Sender<ConnectionHandlerRequest>,
+    dial_request_tx: channel::Sender<TransportRequest>,
     /// Sender for connection events.
-    connection_notifs_tx: channel::Sender<ConnectionNotification<TSocket>>,
+    connection_notifs_tx: channel::Sender<TransportNotification<TSocket>>,
     /// Receiver for connection requests.
     connection_reqs_rx: libra_channel::Receiver<PeerId, ConnectionRequest>,
     /// Receiver for connection events.
-    connection_notifs_rx: channel::Receiver<ConnectionNotification<TSocket>>,
+    connection_notifs_rx: channel::Receiver<TransportNotification<TSocket>>,
```
Comment (phlip9) [inline network/src/peer_manager/mod.rs:248]: transport_notifs_rx

File: network/src/peer_manager/mod.rs [hunk 7]
```diff
@@ -239,13 +239,13 @@ where
     /// Channels to send NewPeer/LostPeer notifications to.
     connection_event_handlers: Vec<conn_status_channel::Sender>,
     /// Channel used to send Dial requests to the ConnectionHandler actor
-    dial_request_tx: channel::Sender<ConnectionHandlerRequest>,
+    dial_request_tx: channel::Sender<TransportRequest>,
     /// Sender for connection events.
-    connection_notifs_tx: channel::Sender<ConnectionNotification<TSocket>>,
+    connection_notifs_tx: channel::Sender<TransportNotification<TSocket>>,
```
Comment (phlip9) [inline network/src/peer_manager/mod.rs:244]: transport_notifs_tx

**Pull request closed without merging**